example /main.py

```diff
@@ -0,0 +1,150 @@
+class business:
```

Comment:
class names must be "camel case": AnExampleName

not lower case: anexamplename
or snale case: an_example_name